src/main/java/org/linecode/server/Position.java

```diff
@@ -0,0 +1,55 @@
+/*
+ * PORTACS
+ * piattaforma di controllo mobilità autonoma
+ *
+ * Copyright lineCode group <linecode.swe@gmail.com> 2020 - 2021
+ * Distributed under open-source licence (see accompanying file LICENCE).
+ */
+
+
+package org.linecode.server;
+
+public class Position {
+    private final int x,y;
+
+    public Position() {
+        this.x=0;
+        this.y=0;
+    }
+    public Position(int x, int y){
+        this.x=x;
+        this.y=y;
+    }
+    public Position(String position) { //non so se va bene questa cosa
+        int virgola=position.indexOf(",");
+        this.x=Integer.parseInt(position.substring(1,virgola-1));
+        this.y=Integer.parseInt(position.substring(virgola+1,position.length()-2));
+    }
```

Comment:
Per rispondere al commento di riga 23: no, non va bene e non è riportato in architettura

Reply:
Era rimasto dal poc credo

Reply:
Okay nvm, è una cosa che dovrebbe aver aggiunto Paolo, bisogna parlarne con lui